traptor/traptor.py

```diff
@@ -1318,6 +1334,12 @@ def run(self, args=None, aLogger=None):
 
             # Concatenate all of the rule['value'] fields
             self.twitter_rules = self._make_twitter_rules(self.redis_rules)
+
+            if len(self.twitter_rules) == 0:
+                self.logger.info("No rules returned")
```

Comment:
Lets update this line:
```
             self.logger.warn('No valid Redis rules assigned', extra=logExtra({
                     'sleep_seconds': self.rule_check_interval
             }))
```
The time this would ever be invoked is going to be super rare.

Reply:
@andrewkcarter adjusted it. And yeah - in a functional prod environ that message should basically never get hit.